airflow/configuration.py

```diff
@@ -580,10 +589,24 @@ def getint(self, section, key):
     def getfloat(self, section, key):
         return float(self.get(section, key))
 
+    def remove_option(self, section, option):
+        try:
+            ConfigParser.remove_option(self, section, option)
+        except ConfigParser.NoSectionError:
+            pass
+        try:
+            ConfigParser.remove_option(self.defaults, section, option)
+        except ConfigParser.NoSectionError:
+            pass
+
     def read(self, filenames):
         ConfigParser.read(self, filenames)
         self._validate()
 
+    def has_option(self, section, option):
```

Comment:
Defining option is done explicitly by users.
Thus could `remove_option` be done also explicitly? 

* `conf.remove_option('something')`
* `conf.remove_option('something_cmd')`
* In case of env option, it could be removed `del os.environ[something]`